src/operator/account/AccountViewHeader.tsx

```diff
@@ -59,21 +61,26 @@ const AccountViewHeader: FC = () => {
           Convert to Contract
         </ButtonBase>
       )}
-      {hasWritePermissions && (
+      {hasWritePermissions && account?.deletable && (
         <ButtonBase
           color={ComponentColor.Danger}
           shape={ButtonShape.Default}
           onClick={() => setDeleteOverlayVisible(!deleteOverlayVisible)}
-          status={
-            account?.deletable
-              ? ComponentStatus.Default
-              : ComponentStatus.Disabled
-          }
           testID="account-delete--button"
         >
           Delete Account
         </ButtonBase>
       )}
+      {hasWritePermissions && account?.cancellable && (
+        <ButtonBase
+          color={ComponentColor.Danger}
+          shape={ButtonShape.Default}
+          onClick={() => setCancelOverlayVisible(!cancelOverlayVisible)}
```

Comment:
Same here- do we need the bang operator when we can just set `setCancelOverlayVisible` to true, since the button shows the overlay rather than toggling it?